components/Main/Exercises/Exercises.js

```diff
@@ -26,44 +26,92 @@ export default function Exercises() {
 
   return (
     <StyledExercises>
-      <h2>Biceps Exercises</h2>
-      <SearchBar onChange={handleChange} setSearchTerm={setSearchTerm} />
-      {dataBiceps
-        .filter((exercise) => {
-          if (searchTerm === "") {
-            return exercise;
-          } else if (
-            exercise.name.toLowerCase().includes(searchTerm.toLowerCase())
-          ) {
-            return exercise;
-          }
-        })
-        .map(({ name, equipment, difficulty, instructions, id }) => (
-          <li key={id}>
-            <div className="favContainer">
-              <button
-                onClick={() => {
-                  handleToggle(id);
-                }}
-              >
-                <span>
-                  <h3>{name.toUpperCase()}</h3>
-                </span>
-              </button>
-              <Favorite />
-            </div>
-            {toggledExercises.includes(id) ? (
-              <Description
-                key={id}
-                id={id}
-                name={name}
-                equipment={equipment}
-                difficulty={difficulty}
-                instructions={instructions}
-              />
-            ) : null}
-          </li>
-        ))}
+      {!isFavouriteExercise ? (
+        <>
+          <h2>Biceps Exercises</h2>
+          <SearchBar onChange={handleChange} setSearchTerm={setSearchTerm} />
+          {dataBiceps
+            .filter((exercise) => {
+              if (searchTerm === "") {
+                return exercise;
+              } else if (
+                exercise.name.toLowerCase().includes(searchTerm.toLowerCase())
+              ) {
+                return exercise;
+              }
+            })
+            .map((exercise) => (
+              <li key={exercise.id}>
+                <div
+                  className="favContainer"
+                  onClick={() => {
+                    handleToggle(exercise.id);
+                  }}
+                >
+                  <button>
+                    <span>
+                      <h3>{exercise.name.toUpperCase()}</h3>
+                    </span>
```

Comment:
Do you need the span? I think the h3 is enough :)

Reply:
deleted it. Thank you :)